apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.html

```diff
@@ -654,7 +654,7 @@
                 </p>
                 <p class="abc " [ngStyle]="{ 'font-size': inputTemplate.fontMedium + 'px', 'white-space' : 'pre-line'} "
                     *ngIf="fieldsAndVisibility.footer.companyAddress?.display ">
-                    {{fieldsAndVisibility.footer.companyAddress?.label ? fieldsAndVisibility.footer.companyAddress?.label : '405-406, Capt. C. S. Naydu Arcade, 10/2, Old Palasia, near Greater Kailash Hospital, Indore 452001(M. P.)'}}
```

Comment:
![image](https://user-images.githubusercontent.com/8776031/72888711-1352ab00-3d34-11ea-93fe-3597fd474d0a.png)

We can merge this if they say they don't want to fix that in other places.

Reply:
it is hardcoded in templates and magic link only
n template this hardcoded data is used for preview templates to used as demo
this will show user how selected template will look like